### DISCUSSION TITLE
Enh prompt management system

### DISCUSSION THREAD

**Discussion by flefevre** [langfuse/langfuse#4276] [Ideas]
### Describe the feature or potential improvement

I would like to know if langfuse will enhance the different features linked to to 'prompt management system '. They are different platforms that are going to this direction. What is the vision of the admin co-founder team of langfuse?

**Comment by marcklingen** [comment]
If you can join, let's discuss the roadmap and ideas for the prompt management feature during this week's townhall.

https://lu.ma/c7zsbc3b

We can afterwards update the thread here.
